README.md

```diff
@@ -207,7 +207,69 @@ $ cat /home/jeanmi/flag
 
 ### Step 4
 
-> WIP
+Now that you are able to execute commands on the server, you can gain an access with a reverse shell.
```

Comment:
You actually don’t NEED it. I don’t see why you couldn’t do the rest without the reverse shell. That’s very basic comfort I’ll admit it. But please tell it.